### DISCUSSION TITLE
TalkNet Demo

### DISCUSSION THREAD

**Discussion by ZhengRachel** [NVIDIA-NeMo/NeMo#3163] [Q&A]
I've just finished reading the paper of TalkNet and I'm interested in the work.
But I'm wondering where I can find some TalkNet Demos. 
Thanks in advance for your answers !

**Comment by blisc** [comment]
@Oktai15 @stasbel Can you suggest some resources?

**Comment by stasbel** [comment]
@ZhengRachel Hi, please, wait #3092 to be merged. There are some subtle bug which prevents TalkNet loading.
I will update you soon on the progress.
